### DISCUSSION TITLE
aspnet-mvc-bridging-the-gap-with-php-development/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#29] [Blog comments]
# aspnet-mvc-bridging-the-gap-with-php-development/

https://blogs.taiga.nl/martijn/posts/aspnet-mvc-bridging-the-gap-with-php-development/

**Comment by disqus-comments-migrator** [comment]
<em>Nico Jansen commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at mei 30 2009, 12:28</em>

---

<p>Hi Martijn,</p><p>As usual you are right. At the Microsoft DevDay's in the Hague there was a php stand.</p><p>Greetings from Capelle aan den Ijssel</p>
